kornia/filters/unsharp.py

```diff
@@ -0,0 +1,79 @@
+from typing import Tuple
+import numpy as np
+import torch
+import torch.nn as nn
+
+import kornia
+from kornia.filters import GaussianBlur2d
+
+def sharpen(
+        input: np.ndarray,
+        kernel_size: Tuple[int, int],
+        sigma: Tuple[float, float],
+        border_type: str = 'reflect') -> np.ndarray:
+    r"""Creates an operator that blurs a tensor using the existing Gaussian filter available with the Kornia library.
+
+
+    Arguments:
+        input (np.ndarray): the input image.
+        kernel_size (Tuple[int, int]): the size of the kernel.
+        sigma (Tuple[float, float]): the standard deviation of the kernel.
+        border_type (str): the padding mode to be applied before convolving.
+          The expected modes are: ``'constant'``, ``'reflect'``,
+          ``'replicate'`` or ``'circular'``. Default: ``'reflect'``.
+
+    Returns:
+        np.ndarray: the sharpened image.
+
+    Examples:
+        >>> input = cv2.imread(path_to_image)
+        >>> output = sharpen(input, (3, 3), (1.5, 1.5))
+        >>> type(output)
+        <class 'numpy.ndarray'>
+    """
+    data: torch.tensor = kornia.image_to_tensor(input, keepdim=False)
+    data=data.float()/255.
+    gauss = kornia.filters.GaussianBlur2d(kernel_size, sigma)
```

Comment:
gauss does not have typing

Reply:
I would even use the functional for consistency `gaussian_blur2d`. The from the top import as `from kornia.filters import gaussian_blur2d`

Reply:
gauss is an operator that initializes the Gaussian Blur with parameters right? So do we need typing?